README.md

```diff
@@ -29,29 +30,34 @@ a specific cloud.
             <rootpw>p@55word!</rootpw>
         </os>
     </template>
+```
 
 Ensure to change the element to your desired root password.
 
 Next, use the imagefactory command and specify the template to use and for which
 clouds to build an image. The above template example was saved to a file name f12_64.tdl.
 
+```bash
     $ sudo imagefactory --template f12_64.tdl --target ec2
```

Comment:
This is not valid bash code with the prompt `$` included, IMHO it would be nicer to remove the prompt. Everyone who understands what it should mean can probably also deduct this from `sudo` being used.